### DISCUSSION TITLE
Package uploaded from public repo appears as Private

### DISCUSSION THREAD

**Discussion by David-Lor** [community/community#22009] [New to GitHub]
<p>I’ve created a fork of a public repository, as a public fork repository. I modified some stuff and <a href="https://github.com/David-Lor/socks-proxy-balancer/blob/6b4a6db9a5d7b4585c68f625585889b5c7759e4e/.github/workflows/build-test-push.yaml" rel="noopener nofollow ugc">created a Github Actions workflow</a> that builds a Docker image and publishes it on the Github Container Registry. However, the uploaded image appears as Private, so only I can see and download it.</p>
<p>Is there any way to upload packages to my fork as Public? Or shall I create a normal repo instead?</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>A container image is private by default at first, you can set it to public from there. “Visibility” is the part you need, “access control” is for when you want to grant access to certain people:</p>
<aside class="onebox allowlistedgeneric" data-onebox-src="https://docs.github.com/en/packages/learn-github-packages/configuring-a-packages-access-control-and-visibility">
  <header class="source">
      <img src="https://docs.github.com/assets/cb-803/images/site/favicon.svg" class="site-icon" width="16" height="16">

      <a href="https://docs.github.com/en/packages/learn-github-packages/configuring-a-packages-access-control-and-visibility" target="_blank" rel="noopener">GitHub Docs</a>
  </header>

  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/5814919/181068228-8a25c7a8-ad81-4876-b29c-630b018ac591.png" class="thumbnail onebox-avatar" width="500" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_500x500.png, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_750x750.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_1000x1000.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_10x10.png">

<h3><a href="https://docs.github.com/en/packages/learn-github-packages/configuring-a-packages-access-control-and-visibility" target="_blank" rel="noopener">Configuring a package's access control and visibility - GitHub Docs</a></h3>

  <p>Choose who has read, write, or admin access to your container image and the visibility of your container images on GitHub.</p>


  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

**Comment by David-Lor** [comment]
<p>Thank you, Finally found it. At first I did not notice the “Package settings” section on individual packages, I thought it had to be configured on Repository-level settings. The docs were a bit confusing, nevertheless I found the “Package settings” by entering the package (container image) I uploaded on my fork repo.</p>
<p>I’ve been uploading more packages before and they were always uploaded Public by default, that’s why I was wondering if it may have something to do with being a fork, or maybe the fork inherited the visibility settings from the original repo.</p>
